### ISSUE TITLE
Links to files contain double "//" and thus not work in the details report when using istanbul

### ISSUE THREAD

**Issue by sandinosaso** [MishaKav/jest-coverage-comment#22]
Problem:

- When using istanbul to generate the reports (and not jest directly) because we are running tests in parallel and then merging the files using istanbul the text report that is saved to file contains for folder always the character "/" at the end, this makes that when seeing the links in Markdowns those not work because there is a double "/"
- I found this happens on parseCoverage function where we always concatenate the "/" without matter of it end with it or not 
```
if (!isCurrentFolder && folders.length) {
      parsedLine[0] = `${folders.at(-1)}/${parsedLine.at(0)}`
    }
```

One solution would be only add that if not `folders.at(-1).endsWith('/')` I did that and then got a bunch of tests failing, I think there are more places where you define what a File/Folder is maybe with a little of help from authors I can help to find a solution, or if you have any ideas how to fix this it would be really cool.

Thank you  🙇  for this nice tool very handy I love it. keep the great work!! 🦸 

Difference between jest text output vs istanbul text output:

![Screenshot 2022-08-09 at 14 17 09](https://user-images.githubusercontent.com/466105/183697241-8d68c4e2-50aa-4e14-bc65-0a643d601a81.png)

Example how I run this to merge multiple json files into one, also for text and text-summary:

![Screenshot 2022-08-09 at 17 37 26](https://user-images.githubusercontent.com/466105/183697247-9ffc4052-248d-40d9-8c64-78c57957a028.png)

The file where the bug happens:
![Screenshot 2022-08-09 at 17 38 52](https://user-images.githubusercontent.com/466105/183697250-15ef55b7-0ec6-4058-b98c-722921d7ddb7.png)

**Comment by MishaKav** [comment]
Can you provide me some of your `output` as `text`, so I can run some tests on it?

**Comment by MishaKav** [comment]
I've implemented this feature in `v1.0.13`.
Try to use the latest version 
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@main
```
or 
```yaml
- name: Jest coverage comment
  uses: MishaKav/jest-coverage-comment@v1.0.13
```

**Comment by sandinosaso** [comment]
Thank you it is working as expected 🙇
